README.md

```diff
@@ -21,7 +21,7 @@ These instructions will get you a copy of the role for your ansible playbook. On
 
 ### Prerequisities
 
-Ansible 2.2.0.0 version installed.
+Ansible 2.2.1.0 version installed.
```

Comment:
Taking advance of this PR, Could yo change the path of the logo (using absolute path instead of relative path)?